### PR TITLE
feat(amf): Support for default-static QoS configuration

### LINKED ISSUE #8966: [AMF]: To support default QoS profile from subscriberd
AMF fetch the default QoS attributes such as pDUSessionAggregateMaximumBitRateDL and pDUSessionAggregateMaximumBitRateUL from Subscriberdb using s6a_proxy grpc

### PR DESCRIPTION
**Pull request by Kaleem-Wavelabs**
Fix: 
1) To fetch default QoS on S6a interface
2) Transfer default QoS to sessiond
3) Set the latest QoS from sessiond

Test:
1) With UE simulator
2) Orchestrator

## Additional Information

- [ ] This change is backwards-breaking

### CODE REVIEW

File: lte/protos/session_manager.proto [hunk 1]
```diff
@@ -852,6 +852,21 @@ enum SMSessionFSMState {
...
+   uint64 br_ul = 1;
+   uint64 br_dl = 2;
+
+   enum BitrateUnitsAMBR {
+         BPS = 0;
+         KBPS = 1;
+         }
+   // Unit (either bps or Kbps)
+   BitrateUnitsAMBR br_unit = 3;
+
```
Comment (themarwhal) [inline lte/protos/session_manager.proto]: spacing looks weird here.

Don't we already have an ambr struct somewhere else?

Comment (Kaleem-Wavelabs) [reply lte/protos/session_manager.proto]: > spacing looks weird here.
> 
> Don't we already have an ambr struct somewhere else?
Fixed the unwanted spaces in proto file
Similar structure Ambr is present under oai package, not able to include oai package under session_manager.proto, so separately defined it

File: lte/protos/session_manager.proto [hunk 2]
```diff
@@ -852,6 +852,17 @@ enum SMSessionFSMState {
...
+message Ambr {
+  uint64 br_ul = 1;
+  uint64 br_dl = 2;
+  enum BitrateUnitsAMBR {
+        BPS = 0;
+        KBPS = 1;
+  }
+  //Unit (either bps or Kbps)
+  BitrateUnitsAMBR br_unit = 3;
+}
```
Comment (ulaskozat) [inline lte/protos/session_manager.proto]: isn't this the same definition as `message AggregatedMaximumBitrate` in apn.proto? Please reuse existing proto definitions.

Comment (Kaleem-Wavelabs) [reply lte/protos/session_manager.proto]: Fixed the same. Thanks

**Review by ulaskozat — APPROVED** [review]
LGTM

**Pull request merged**